### PR TITLE
Add Chinese PSENet model trained on CTW1500

### PR DESCRIPTION
**Pull request by VictorHe-1**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

Add PSENet model trained on CTW1500

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: configs/det/psenet/README.md [hunk 1]
```diff
@@ -32,6 +32,14 @@ The overall architecture of PSENet is presented in Figure 1. It consists of mult
...
+
+| **Model**              | **Context**       | **Backbone**      | **Pretrained** | **Recall** | **Precision** | **F-score** | **Train T.**     | **Throughput**   | **Recipe**                            | **Download**                                                                                                                                                                                                |
+|---------------------|----------------|---------------|------------|------------|---------------|-------------|--------------|-----------|-------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| PSENet               | D910x8-MS2.0-G | ResNet-152   | ImageNet   | 67.41%     | 78.78%        | 72.65%      | 67 s/epoch   | 14.92 img/s | [yaml](pse_r152_ctw1500.yaml) | [ckpt](https://download.mindspore.cn/toolkits/mindocr/psenet/psenet_resnet152_ctw1500-756e9896.ckpt) \| [mindir](https://download.mindspore.cn/toolkits/mindocr/psenet/psenet_resnet152_ctw1500-756e9896-bbd8cc51.mindir)
```
Comment (Songyuanwei) [inline configs/det/psenet/README.md:40]: ctw1500的f-score只有72.65吗？icdar有82.06，这两个数据集的差距不应该有这么大吧。。

File: configs/det/psenet/pse_r152_ctw1500.yaml [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    hidden_size: 256
+    out_channels: 7
+
+
+postprocess:
+  name: PSEPostprocess
+  binary_thresh: 0
+  box_thresh: 0.85
+  min_area: 16
+  box_type: 'quad'
```
Comment (Songyuanwei) [inline configs/det/psenet/pse_r152_ctw1500.yaml:29]: ctw1500数据集是弯曲文本，如果psenet可以处理弯曲文本的话，`box_type`应该是`poly`而不是`quad`，这样f-score应该可以提高很多

Comment (VictorHe-1) [reply configs/det/psenet/pse_r152_ctw1500.yaml:29]: 好的，我试下看看

File: configs/det/psenet/README.md [hunk 3]
```diff
@@ -67,6 +75,26 @@ The prepared dataset file struture should be:
     └── train_det_gt.txt
 ```
 
+#### 3.2.2 SCUT-CTW1500 dataset
+
+Please download [SCUT-CTW1500](https://github.com/Yuliang-Liu/Curve-Text-Detector) dataset，and convert the labels to the desired format referring to [dataset_converters](https://github.com/mindspore-lab/mindocr/blob/main/tools/dataset_converters/README.md).
```
Comment (HaoyangLee) [inline configs/det/psenet/README.md:80]:
```suggestion
Please download [SCUT-CTW1500](https://github.com/Yuliang-Liu/Curve-Text-Detector) dataset, and convert the labels to the desired format referring to [dataset_converters](https://github.com/mindspore-lab/mindocr/blob/main/tools/dataset_converters/README.md).
```

File: configs/det/psenet/README.md [hunk 4]
```diff
@@ -32,6 +32,14 @@ The overall architecture of PSENet is presented in Figure 1. It consists of mult
...
+
+| **Model**              | **Context**       | **Backbone**      | **Pretrained** | **Recall** | **Precision** | **F-score** | **Train T.**     | **Throughput**   | **Recipe**                            | **Download**                                                                                                                                                                                                |
+|---------------------|----------------|---------------|------------|------------|---------------|-------------|--------------|-----------|-------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| PSENet               | D910x8-MS2.0-G | ResNet-152   | ImageNet   | 73.69%     | 74.38%        | 74.04%      | 67 s/epoch   | 14.33 img/s | [yaml](pse_r152_ctw1500.yaml) | [ckpt](https://download.mindspore.cn/toolkits/mindocr/psenet/psenet_resnet152_ctw1500-58b1b1ff.ckpt) \| [mindir](https://download.mindspore.cn/toolkits/mindocr/psenet/psenet_resnet152_ctw1500-58b1b1ff-b95c7f85.mindir)
```
Comment (HaoyangLee) [inline configs/det/psenet/README.md:40]: 现在readme中有两个mindir链接，在Notes中说明这两个mindir链接的导出shape**分别**是多少。

Comment (VictorHe-1) [reply configs/det/psenet/README.md:40]: 好的

File: configs/det/psenet/README.md (hunk 3 above)
Comment (VictorHe-1) [reply configs/det/psenet/README.md:80]: ok

**Review by HaoyangLee — APPROVED** [review]

**Review by liangxhao — APPROVED** [review]

**Pull request merged**